lwgenerate/azure/azure.go

```diff
@@ -0,0 +1,250 @@
+package azure
+
+import (
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/lacework/go-sdk/lwgenerate"
+	"github.com/pkg/errors"
+)
+
+type GenerateAzureTfConfigurationArgs struct {
```

Comment:
I would expect this struct to mirror (closely, likely not exactly) the options found...

here: https://registry.terraform.io/modules/lacework/activity-log/azure/latest#inputs

and here: https://registry.terraform.io/modules/lacework/config/azure/latest#inputs

Or are we handling that another way?

Reply:
➕1 on adding the options from terraform registry inputs ☝️. This is what has been done for GCP

Reply:
Yeah I have updated this to mirror more of the options that we require, but still able to create the 'no touch' terraform as per the help files example if the user does not set any config